app/searches/lookup_drugs.rb

```diff
@@ -29,8 +30,9 @@ def self.match_search_terms_to_objects(search_terms, scope)
     search_terms = search_terms.dup
     drug_results = DataModel::Drug.send(scope).where(name: search_terms)
     search_terms = search_terms - drug_results.map(&:name)
-    drug_alias_results = DataModel::DrugClaimAlias.send(scope).where(alias: search_terms)
+    drug_alias_results = DataModel::DrugAlias.send(scope).where(alias: search_terms)
     search_terms = search_terms - drug_alias_results.map(&:alias)
+    #Can we get rid of this case
```

Comment:
I don't think so? We've associated all of the drug claim info we're interested in searching to the drug level--we probably don't need `for_search` in `drug_claim.rb` at all anymore, although I'd be interested to hear @acoffman and @yanyangfeng's thoughts on this.